system-addon/content-src/styles/_theme.scss

```diff
@@ -7,6 +7,17 @@
   --newtab-textbox-focus-boxshadow: textbox-shadow($color);
 }
 
+// scss variables related to the theme.
+$border-primary: 1px solid var(--newtab-border-primary-color);
+$border-secondary: 1px solid var(--newtab-border-secondary-color);
+$shadow-primary: 0 0 0 5px var(--newtab-card-active-outline-color);
+$shadow-secondary: 0 1px 4px 0 $grey-90-20;
+$input-border: solid 1px var(--newtab-textbox-border);
+$input-border-active: solid 1px var(--newtab-textbox-focus-color);
+$input-error-border: solid 1px $red-60;
+$input-error-boxshadow: textbox-shadow($red-60);
+$inner-box-shadow: 0 0 0 1px var(--newtab-inner-box-shadow-color);
```

Comment:
I suppose while we're touching these... ;) Sort the list alphabetically and make the borders consistently `1px solid <color>`